src/Mimi.jl

```diff
@@ -2,14 +2,15 @@ module Mimi
 
 include("metainfo.jl")
 include("clock.jl")
+include("graph_API.jl")
 using DataStructures
 using DataFrames
 using Distributions
 
 export
     ComponentState, run_timestep, run, @defcomp, Model, setindex, addcomponent, setparameter,
     connectparameter, setleftoverparameters, getvariable, adder, MarginalModel, getindex,
-    getdataframe, components, variables, setbestguess, setrandom, getvpd, unitcheck
+    getdataframe, components, variables, setbestguess, setrandom, getvpd, unitcheck, print_graph
```

Comment:
Don't export anything, instead we overwrite the ``show`` method as discussed yesterday.